includes/Checker/Checks/Performance/Non_Blocking_Scripts_Check.php

```diff
@@ -0,0 +1,243 @@
+<?php
+/**
+ * Class Non_Blocking_Scripts_Check
```

Comment:
```suggestion
 * Class Non_Blocking_Scripts_Check.
```

Add end period per WPCS.